resetpassword/lib/__init__.py

```diff
@@ -58,16 +59,22 @@ def _plain_send_mail(sender, recipient, subject, body):
     sender_name, sender_addr = parseaddr(sender)
     recipient_name, recipient_addr = parseaddr(recipient)
 
-    sender_name = str(Header(unicode(sender_name), header_charset))
-    recipient_name = str(Header(unicode(recipient_name), header_charset))
+    sender_name = str(Header(text(sender_name), header_charset))
```

Comment:
This changes the meaning of the code quite a lot.
When `unicode` strings are provided to `Header` the second parameters expresses how they should be encoded when sending them out, while when `bytes` are provided it expresses the current encoding of the text inside the bytes.

The proposed patch would make such code behave in two different ways on Py2 and PY3 as in Py3 it would use `header_charset` as the outgoing charset, while on Py2 it would use `header_charset` as the charset used to decode the `sender_name`.

Same applies below for `recipient_name`.

to retain previous behaviour sender_name must always be an `unicode` on both py2 and py3, so it needs to be `unicode` on py2 and `str` on py3.

Something like:

```
try:
    unicode_text = unicode
except NameError:
    unicode_text = str 
```

should cope with such case.